doc/sphinx/addendum/type-classes.rst

```diff
@@ -582,9 +582,9 @@ Typeclasses eauto `:=`
    + ``debug`` In debug mode, the trace of successfully applied tactics is
      printed.  This value can also be set with :flag:`Typeclasses Debug`.
 
-   + ``dfs, bfs`` This sets the search strategy to depth-first search (the
+   + ``(dfs)``, ``(bfs)`` This sets the search strategy to depth-first search (the
      default) or breadth-first search.  This value can also be set with
```

Comment:
Maybe change "This value" -> "The search strategy".  IHMO, a "search strategy" is a setting that has a value--but is not itself a value.